edge-modules/TestAnalyzer/PartitionReceiverHandler.cs

```diff
@@ -15,7 +15,7 @@ class PartitionReceiveHandler : IPartitionReceiveHandler
         const string SequenceNumberPropertyName = "sequenceNumber";
         const string EnqueuedTimePropertyName = "iothub-enqueuedtime";
         const string BatchIdPropertyName = "batchId";
-        static readonly ILogger Log = Logger.Factory.CreateLogger<PartitionReceiveHandler>();
+        static readonly ILogger Logger = ModuleUtil.CreateLogger("Analyzer");
```

Comment:
why change to use Analayzer instead of PartitionReceiverHandler.

Reply:
I thought the pattern was to have the name of the module, not the class, but I am wrong. I'll change.